cmd/fyne/internal/commands/package.go

```diff
@@ -376,10 +376,6 @@ func calculateExeName(sourceDir, os string) string {
 
 	if os == "windows" {
 		exeName = exeName + ".exe"
-	} else if os == "wasm" {
```

Comment:
is it OK that this code is removed?

Reply:
Yes, I have moved the logic to add the extension in package-web.go where it is actually making more sense as it match the case where we are generating two executable and need to call build appropriately.